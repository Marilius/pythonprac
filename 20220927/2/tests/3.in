12,41,51,12,41,51,12,41,51