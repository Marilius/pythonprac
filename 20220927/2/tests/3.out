[51, 51, 51, 12, 12, 12, 41, 41, 41]